workers/unity/Packages/com.improbable.gdk.core/Systems/WorkerSystem.cs

```diff
@@ -27,6 +27,12 @@ public WorkerSystem(Connection connection, ILogDispatcher logDispatcher, string
             Origin = origin;
         }
 
+        public static WorkerSystem Instantiate(World world,
```

Comment:
Why add this?

Reply:
The main benefit of using named parameters instead of relying on the `params object[]` via `CreateManager` is type checking at compile time rather than runtime.

This way, if we want to add another parameter or adjust the constructor in another way, there would be runtime problems whose discovery would be more difficult when compared to this.

Reply:
You still have this issue now? Since you would add the parameter to the constructor AND the `Instantiate` function.
Just moving the problem a layer deeper. I'm rather against this change.

Reply:
Let's say you update the constructor without this change; you need to manually find all places where you have called `CreateManager<WorkerSystem>(...)` and ensure you have modified them appropriately. If you miss a few, you won't get compile problems, but it may get triggered when the piece of code gets executed.

With this change, you modify the constructor and `Instantiate`, and all the places that use it have to be fixed until the user code can compile again.

However, since this code is used in the unconditionally for all execution paths that use `WorkerSystem`, the errors should be caught immediately when you run anything of that sort.

Therefore I'm happy to revert this.